contracts/Everlong.sol

```diff
@@ -417,16 +520,18 @@ contract Everlong is IEverlong {
         if (_portfolio.totalBonds != 0) {
             // NOTE: The maturity time is rounded to the next checkpoint to
             // underestimate the portfolio value.
-            value += IHyperdrive(hyperdrive).previewCloseLong(
-                asBase,
-                IEverlong.Position({
-                    maturityTime: IHyperdrive(hyperdrive)
-                        .getCheckpointIdUp(_portfolio.avgMaturityTime)
-                        .toUint128(),
-                    bondAmount: _portfolio.totalBonds
-                }),
-                ""
-            );
+            value += IHyperdrive(hyperdrive)
+                .previewCloseLong(
+                    asBase,
+                    IEverlong.Position({
+                        maturityTime: IHyperdrive(hyperdrive)
+                            .getCheckpointIdUp(_portfolio.avgMaturityTime)
+                            .toUint128(),
+                        bondAmount: _portfolio.totalBonds
+                    }),
+                    ""
+                )
+                .mulDown(1e18 - maxCloseLongSlippage);
```

Comment:
Why are we underestimating this with a slippage parameter? As I mentioned in another comment, the only possible slippage that could occur is if there are small rounding errors, which would certainly be much smaller than 1% of the trade execution.